tests/Benchmark/ArrayOfBench.php

```diff
@@ -0,0 +1,59 @@
+<?php declare(strict_types = 1);
+
+namespace Orisai\Tests\Benchmark;
+
+use Generator;
+use Orisai\ObjectMapper\Exception\InvalidData;
+use Tests\Orisai\ObjectMapper\Doubles\ArrayOfIntVO;
+use Tests\Orisai\ObjectMapper\Doubles\ArrayOfStringVO;
+use Tests\Orisai\ObjectMapper\Toolkit\ProcessingTestCase;
+use function array_fill;
+
+final class ArrayOfBench extends ProcessingTestCase
```

Comment:
Shouldn't extends phpunit test case, but I see the reason. Will solve that myself, some setup for apps without DIC gonna be helpful anyway 
#8

Reply:
Rebased and rest of the issues resolved :crossed_fingers: